src/main/java/mate/academy/library/service/impl/UserServiceImpl.java

```diff
@@ -35,6 +37,7 @@ public UserResponseDto register(UserRegistrationRequestDto requestDto)
         user.setRoles(roles);
         user.setPassword(passwordEncoder.encode(requestDto.getPassword()));
         user = userRepository.save(user);
+        shoppingCartService.save(user);

```

Comment:
It's better to use cartService.createShoppingCart(user);
where in ShoppingCartServiceImpl
```
@Override
    public void createShoppingCart(User user) {
        ShoppingCart shoppingCart = new ShoppingCart();
        shoppingCart.setUser(user);
        repository.save(shoppingCart);
    }
```